main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -1750,7 +1750,8 @@ object Weeder2 {
       val objName = pickNameIdent(tree)
       mapN(objName, fragments) {
         case (objName, fragments) =>
-          val nameExpr = Expr.Ambiguous(Name.QName(Name.RootNS, objName, objName.loc), objName.loc)
+          //Expr.Ambiguous(Name.mkQName(prefix.map(_.toString), ident.name, ident.loc), ident.loc)
+          val nameExpr = Expr.Ambiguous(Name.mkQName(List(), objName.name, objName.loc), objName.loc)
```

Comment:
Whats this change about?

Reply:
Ah this was a mistake from another branch, sorry.

Reply:
@chanattan No worries. Thats why we do code reviews :)